packages/manager/modules/pci/src/projects/project/training/jobs/info/info.controller.js

```diff
@@ -3,33 +3,61 @@ import { GUIDE_URL } from '../../training.constants';
 
 export default class PciTrainingJobsInfoController {
   /* @ngInject */
-  constructor(CucCloudMessage, CucRegionService, $interval) {
+  constructor(
+    CucCloudMessage,
+    CucRegionService,
+    $interval,
+    PciProjectTrainingJobService,
+  ) {
     this.CucCloudMessage = CucCloudMessage;
     this.CucRegionService = CucRegionService;
     this.$interval = $interval;
+    this.PciProjectTrainingJobService = PciProjectTrainingJobService;
   }
 
   $onDestroy() {
     if (this.interval !== null) {
       this.$interval.cancel(this.interval);
     }
+    if (this.pullInterval !== null) {
+      this.$interval.cancel(this.pullInterval);
+    }
+  }
+
+  // eslint-disable-next-line class-methods-use-this
+  isError(state) {
+    return state === 'FAILED' || state === 'ERROR';
```

Comment:
we should avoid disabling eslint. To avoid this the isError could be moved to a `Job` class if there is one or the method should be static

Reply:
it's not linked to the job but to his history.

Reply:
If elements of the history could also be represented as models it would be great, but if it is not possible then at least we should use a static method

Reply:
so I need to extract a static method in job ?